<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainApp</class>
 <widget class="QMainWindow" name="MainApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1264</width>
    <height>1014</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainApp</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="nameLineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>261</width>
      <height>42</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="phoneNumberLineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>261</width>
      <height>42</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>80</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>550</y>
      <width>103</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="viewAllButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>40</y>
      <width>121</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string>View All</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>130</y>
      <width>671</width>
      <height>691</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="deleteByName">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>650</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>201</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>281</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Phone number</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>610</y>
      <width>221</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Name</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1264</width>
     <height>39</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
